product_docs/docs/epas/15/database_administration/02_edb_loader/data_loading_methods.mdx

```diff
@@ -6,19 +6,12 @@ description: "Description of the data loading methods supported by EDB*Loader"
 
 As with Oracle SQL\*Loader, EDB\*Loader supports three data loading methods:
 
--   Conventional path load
--   Direct path load
--   Parallel direct path load
+- *Conventional path load* &mdash; Conventional path load is the default method used by EDB\*Loader. Use basic insert processing to add rows to the table. The advantage of a conventional path load is that table constraints and database objects defined on the table are enforced during a conventional path load. Table constraints and database objects include primary keys, not null constraints, check constraints, unique indexes, foreign key constraints, triggers, and so on. One exception is that the EDB Postgres Advanced Server rules defined on the table aren't enforced. EDB\*Loader can load tables on which rules are defined. However, the rules aren't executed. As a consequence, you can't load partitioned tables implemented using rules with EDB\*Loader. See [Conventional path load](invoking_edb_loader/conventional_path_load.mdx)
```

Comment:
Can we have either all namings in singular or plural. This one has "load" at the end whereas other two methods says "loads" at the end